install.md

```diff
@@ -28,18 +34,107 @@ This guide will walk you through the installation of [CRI-O](https://github.com/
+
+| Operating system | $OS               |
+| ---------------- | ----------------- |
+| Centos 8         | `CentOS_8`        |
+| Centos 8 Stream  | `CentOS_8_Stream` |
+| Centos 7         | `CentOS_7`        |
+
+
+And then run the following as root:
+```shell
+curl -L -o /etc/yum.repos.d/devel:kubic:libcontainers:stable.repo https://download.opensuse.org/repositories/devel:/kubic:/libcontainers:/stable/$OS/devel:kubic:libcontainers:stable.repo
+curl -L -o /etc/yum.repos.d/devel:kubic:libcontainers:stable:cri-o:$VERSION.repo https://download.opensuse.org/repositories/devel:kubic:libcontainers:stable:cri-o:$VERSION/$OS/devel:kubic:libcontainers:stable:cri-o:$VERSION.repo
+yum install cri-o
+```
+
+#### APT based operating systems
+
+Note: this tutorial assumes you have curl and gnupg installed
+
+To install on the following operating systems, set the environment variable $OS as the appropriate field in the following table:
+| Operating system | $OS               |
+| ---------------- | ----------------- |
+| Debian Unstable  | `Debian_Unstable` |
+| Debian Testing   | `Debian_Testing`  |
+| Ubuntu 20.04     | `xUbuntu_20.04`   |
+| Ubuntu 19.10     | `xUbuntu_19.10`   |
+| Ubuntu 19.04     | `xUbuntu_19.04`   |
+| Ubuntu 18.04     | `xUbuntu_18.04`   |
+
+And then run the following as root:
+```shell
+echo "deb https://download.opensuse.org/repositories/devel:/kubic:/libcontainers:/stable/$OS/ /" > /etc/apt/sources.list.d/devel:kubic:libcontainers:stable.list
+echo "deb http://download.opensuse.org/repositories/devel:/kubic:/libcontainers:/stable:/cri-o:/$VERSION/$OS/ /" > /etc/apt/sources.list.d/devel:kubic:libcontainers:stable:cri-o:$VERSION.list
+
+curl -L https://download.opensuse.org/repositories/devel:kubic:libcontainers:stable:cri-o:$VERSION/$OS/Release.key | apt-key add -
+curl -L https://download.opensuse.org/repositories/devel:/kubic:/libcontainers:/stable/$OS/Release.key | apt-key add -
+
+apt-get update
+apt-get install cri-o cri-o-runc
+```
+
+Note: We include cri-o-runc because Ubuntu and Debian include their own packaged version of runc.
+While this version should work with CRI-O, keeping the packaged versions of CRI-O and runc in sync ensures they work together.
+If you'd like to use the distribution's runc, you'll have to add the file:
+```toml
+[crio.runtime.runtimes.runc]
+runtime_path = ""
+runtime_type = "oci"
+runtime_root = "/run/runc"
+```
+to `/etc/crio/crio.conf.d/`
+
+## Build and install CRI-O from source
+
+### Runtime dependencies
 
 - runc, Clear Containers runtime, or any other OCI compatible runtime
 - iproute
 - iptables
 
 Latest version of `runc` is expected to be installed on the system. It is picked up as the default runtime by CRI-O.
 
-## Build and Run Dependencies
+### Build and Run Dependencies
 
 
-### Fedora - RHEL 7 - CentOS
+#### Fedora - RHEL 7 - CentOS
 **Required**
 
 Fedora, RHEL<=7, CentOS and related distributions:
```

Comment:
nope, rhel 8 specific instructions are below, so we intend rhel 7 or lower

Reply:
Do we support RHEL-6? Oo

Reply:
... I am not even sure. When we originally wrote these sections, we branched because rhel 8 had changed to dnf and modules, so the process was different. I have not heard of anyone choosing to run rhel 6 with cri-o

Reply:
Yep, ack! It's not even directly related to your series. We can mark this one as resolved!